src/gluonts/evaluation/_base.py

```diff
@@ -101,6 +102,11 @@ class Evaluator:
     chunk_size
         Controls the approximate chunk size each workers handles at a time.
         Default is 32.
+    mask_invalid_timeseries
```

Comment:
Wouldn't `mask_invalid_values` be more accurate name, since we don't mask the entire ts, but just some values?

Reply:
I think that makes sense yes, ill change it.

Reply:
I also change the `nan_if_masked_timeseries` to `nan_if_masked_metric` as this also is more specific.